packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputText from '../../atom/input-text';
+import CheckboxWithTitle from '../../atom/checkbox-with-title';
+import style from './style.css';
+
+const InputTextWithTiteledCheckbox = props => {
+  const {
+    title,
+    name,
+    checked,
+    checkBoxOnChange,
+    checkBoxLabel,
+    inputTextTitle,
+    inputTextValue,
+    inputTextPlaceholder,
+    inputTextDescription,
+    inputTextDisabled,
+    inputTextModified,
+    inputTextOnChange
+  } = props;
+
+  return (
+    <div>
+      <CheckboxWithTitle
+        title={title}
+        name={name}
+        checked={checked}
+        onChange={checkBoxOnChange}
+        data-name="check-box"
+        aria-label={checkBoxLabel}
+      />
+      <div className={style.inputText}>
+        <InputText
+          title={inputTextTitle}
+          value={inputTextValue}
+          placeholder={inputTextPlaceholder}
+          theme="coorpmanager"
+          description={inputTextDescription}
+          disabled={inputTextDisabled}
```

Comment:
on ne lie pas directement l'etat disable au fait que la checkbox soit non coché ?

Reply:
on peut gérer ça côté CM si on veut se laisse plus de maniabilité sur la désactivation mais pour l'instant on a pas de use case

Reply:
add an aria-label for input plz

Reply:
le composant ne prend pas d'aria-label.